README.md

```diff
@@ -5,7 +5,9 @@
 ## Running Locally
 
 This demo is designed to be run outside of a corporate firewall.  The
-moving parts have not been instrumented to run with proxies.
+moving parts have not been instrumented to run with proxies.  This demo
+does not work on the clear-guest WiFi because a key enabler of the demo
```

Comment:
I'd prefer to keep Oracle-specific language out... My goal is to expose this demo wider than just Oracle.